test/controllers/api/v2/organizations_controller_test.rb

```diff
@@ -1,4 +1,5 @@
 require 'test_helper'
+require 'rfauxfactory'
```

Comment:
actually it would be better to place this into test_helper so that every test can use this library